src/components/ShippingDetails/ShippingDetails.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+
+//TODO: Decide about i18n solution or simply a JSON file where we can load translations.
+
+const TRANSLATIONS = {
```

Comment:
For now we haven't decided which solution to use. Keep in mind we've got only 2 weeks to complete whole thing. There is no time for it.
Just set those strings directly.